docker/src/test/resources/Dockerfile

```diff
@@ -14,19 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM ubuntu:14.04
-MAINTAINER Sven Dowideit <SvenDowideit@docker.com>
 
-RUN apt-get update && apt-get install -y openssh-server
-RUN mkdir /var/run/sshd
-RUN echo 'root:screencast' | chpasswd
-RUN sed -i 's/PermitRootLogin without-password/PermitRootLogin yes/' /etc/ssh/sshd_config
 
-# SSH login fix. Otherwise user is kicked off after login
-RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
+FROM alpine:3.2
+MAINTAINER JClouds Dev <jclouds-dev@apache.org>
```

Comment:
fixed

On Thu, Oct 8, 2015 at 11:47 AM, Ignasi Barrera <notifications@github.com>
wrote:

> In docker/src/test/resources/Dockerfile
> <https://github.com/jclouds/jclouds-labs/pull/208#discussion_r41494980>:
>
> >
> > -# SSH login fix. Otherwise user is kicked off after login
> > -RUN sed 's@session\s*required\s*pam_loginuid.so@session optional pam_loginuid.so@g' -i /etc/pam.d/sshd
> > +FROM alpine:3.2
> > +MAINTAINER JClouds Dev <jclouds-dev@apache.org>
>
> [minor] Change to dev@jclouds.apache.org
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jclouds/jclouds-labs/pull/208/files#r41494980>.
>